i18n/si.jaml

```diff
@@ -2010,14 +2010,18 @@ distance/distance.py:
         def `compute_distances`:
             hamming: false
 ensembles/ada_boost.py:
+    def `base_estimator_deprecation`:
+        '`base_estimator` is deprecated (to be removed in 3.39): use `estimator` instead.': false
```

Comment:
This should not be `false`. I think you either provide a translation or leave it `null` for someone else to translate it later.

Reply:
It's OK, this message is not intended to be seen by the user. It should be in English.